contracts/Killable.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity 0.5.10;
+
+import "./SafeMath.sol";
+import "./Owned.sol";
+
+contract Killable is Owned {
+    using SafeMath for uint;
+    
+    uint public constant DELAY = 30 days;
+    
+    struct KillCondition {
+        uint killingDate;
+        bool status;
+    }
+    
```

Comment:
Let's simplify the naming:
1. `bool killed;`
2. `whenAlive()`: A modifier that checks that `killed == false`
3. `whenKilled()`: A modifier that checks that `killed == true`

Reply:
See the last commit. I revamped the Killable contract to meet your feedback. However, I did not find any utility of the whenKilled modifier, so I did not implement it :)